t/math/ambiguous_relations.xml

```diff
@@ -0,0 +1,161 @@
+        <XMath>
+          <XMTok meaning="0" role="NUMBER">0</XMTok>
+          <XMTok meaning="equals" role="RELOP">=</XMTok>
+          <XMTok meaning="less-than" role="RELOP">&lt;</XMTok>
+          <XMTok font="italic" role="UNKNOWN" xml:id="S0.Ex4.m1.1">x</XMTok>
+          <XMTok role="PUNCT">,</XMTok>
+          <XMTok font="italic" role="UNKNOWN" xml:id="S0.Ex4.m1.2">y</XMTok>
+          <XMTok meaning="greater-than" role="RELOP">&gt;</XMTok>
+        </XMath>
+      </Math>
+    </equation>
+  </para>
+  <para xml:id="p5">
+    <equation xml:id="S0.Ex5">
+      <Math mode="display" tex="2&lt;x,y&gt;=z" text="formulae@(2 less x, y &gt;= z)" xml:id="S0.Ex5.m1">
+        <XMath>
+          <XMDual>
+            <XMApp>
+              <XMTok meaning="formulae"/>
+              <XMRef idref="S0.Ex5.m1.3"/>
+              <XMRef idref="S0.Ex5.m1.4"/>
+            </XMApp>
+            <XMWrap>
+              <XMApp xml:id="S0.Ex5.m1.3">
+                <XMTok meaning="less-than" role="RELOP">&lt;</XMTok>
+                <XMTok meaning="2" role="NUMBER">2</XMTok>
+                <XMTok font="italic" role="UNKNOWN" xml:id="S0.Ex5.m1.1">x</XMTok>
+              </XMApp>
+              <XMTok role="PUNCT">,</XMTok>
+              <XMApp xml:id="S0.Ex5.m1.4">
+                <XMTok font="italic" meaning="greater-than-or-equals" role="RELOP">&gt;=</XMTok>
+                <XMTok font="italic" role="UNKNOWN" xml:id="S0.Ex5.m1.2">y</XMTok>
+                <XMTok font="italic" role="UNKNOWN">z</XMTok>
+              </XMApp>
+            </XMWrap>
+          </XMDual>
+        </XMath>
+      </Math>
+    </equation>
+  </para>
+  <para xml:id="p6">
+    <equation xml:id="S0.Ex6">
+      <Math mode="display" tex="0=&lt;x,y&gt;A" xml:id="S0.Ex6.m1">
+        <XMath>
+          <XMTok meaning="0" role="NUMBER">0</XMTok>
+          <XMTok meaning="equals" role="RELOP">=</XMTok>
+          <XMTok meaning="less-than" role="RELOP">&lt;</XMTok>
+          <XMTok font="italic" role="UNKNOWN" xml:id="S0.Ex6.m1.1">x</XMTok>
+          <XMTok role="PUNCT">,</XMTok>
+          <XMTok font="italic" role="UNKNOWN" xml:id="S0.Ex6.m1.2">y</XMTok>
+          <XMTok meaning="greater-than" role="RELOP">&gt;</XMTok>
+          <XMTok font="italic" role="UNKNOWN">A</XMTok>
+        </XMath>
+      </Math>
+    </equation>
+  </para>
+  <para xml:id="p7">
+    <equation xml:id="S0.Ex7">
+      <Math mode="display" tex="a&lt;&lt;b" text="a lessless b" xml:id="S0.Ex7.m1">
+        <XMath>
+          <XMApp>
+            <XMTok font="italic" meaning="much-less-than" role="RELOP">&lt;&lt;</XMTok>
+            <XMTok font="italic" role="UNKNOWN">a</XMTok>
+            <XMTok font="italic" role="UNKNOWN">b</XMTok>
+          </XMApp>
+        </XMath>
+      </Math>
+    </equation>
+  </para>
+  <para xml:id="p8">
+    <equation xml:id="S0.Ex8">
+      <Math mode="display" tex="0&lt;&lt;a,b&gt;&gt;1" text="formulae@(0 lessless a, b &gt;&gt; 1)" xml:id="S0.Ex8.m1">
```

Comment:
I'm a bit confused why the text attribute for `<` is `less`, while the text attribute for `>` is `&gt;`. The resulting value is kind of broken (`lessless`).